### PR TITLE
update github repos to synerbi

### PR DESCRIPTION
**Pull request by paskino**
Updates the URL pointing to CCPPETMR to SynerBI.

### CODE REVIEW

**Comment by paskino** [comment]
Together with https://github.com/SynerBI/CCPPETMR_VM/pull/151 I'm testing that it all works.

**Comment by KrisThielemans** [comment]
All fine. 

Last Travis job "docker +SERVICE +GPU" fails with unrelated problem:
https://travis-ci.org/github/CCPPETMR/SIRF-SuperBuild/jobs/671456370#L4374
```
user_sirf-ubuntu.sh: line 13: cmake: command not found
```
@casperdcl a quick look?

File: CHANGES.md [hunk 1]
```diff
@@ -57,10 +57,10 @@
 
  - optionally build siemens_to_ismrmrd
  - optionally build petmr_rd_tools
- - major [restructuring](https://github.com/CCPPETMR/SIRF-SuperBuild/issues/16#issuecomment-360772097) of SuperBuild build directory
+ - major [restructuring](https://github.com/SynerBI/SIRF-SuperBuild/issues/16#issuecomment-360772097) of SuperBuild build directory
  - User can specify the location of the sources via the variable `SOURCES_ROOT_DIR`
  - Updated dependency versions (STIR)
- - Defaults to `USE_SYSTEM_BOOST=ON` on Windows and MacOSX [#63](https://github.com/CCPPETMR/SIRF-SuperBuild/issues/63), see [here](https://github.com/CCPPETMR/SIRF/wiki/SIRF-SuperBuild-on-MacOS#2-boost-)
+ - Defaults to `USE_SYSTEM_BOOST=ON` on Windows and MacOSX [#63](https://github.com/SynerBI/SIRF-SuperBuild/issues/63), see [here](https://github.com/SynerBI/SIRF/wiki/SIRF-SuperBuild-on-MacOS#2-boost-)
```
[the remark this answers is not in the document]
Comment (KrisThielemans) [reply CHANGES.md:63]: agreed. we are squatting the CCPPETMR org, so let's leave this.

File: NOTICE.txt [hunk 2]
```diff
@@ -1,11 +1,11 @@
-CCP PETMR Synergistic Image Reconstruction Framework (SIRF) SuperBuild.
+CCP SynerBI Synergistic Image Reconstruction Framework (SIRF) SuperBuild.
 Copyright 2017-2019 University College London (UCL)
 Copyright 2017-2019 Rutherford Appleton Laboratory STFC
 Copyright 2017-2019 King's College London (KCL)
```
Comment (KrisThielemans) [inline NOTICE.txt:4]: this needs 2020 added in any case

File: docker/Dockerfile [hunk 3]
```diff
@@ -5,10 +5,10 @@ LABEL \
   author.name="Casper da Costa-Luis"          \
   author.email=imaging@caspersci.uk.to        \
   maintainer.email=ccppetmr@stfc.ac.uk        \
```
Comment (KrisThielemans) [inline docker/Dockerfile:7]: will this email remain ok?

File: cloud/terraform/azure/README.md [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
+Demonstrates how to build and install [SIRF](https://github.com/SynerBI/SIRF) on an [Azure](https://azure.microsoft.com) VM using [Packer](https://www.packer.io/) and [Terraform](https://www.terraform.io/). The VM is described in Packer files. Packer is used to build an image of the VM. Terraform deploys VMs, and their associated infrastructure in the cloud and then performs further configuration. There is currently GPU support (NVIDIA).
 
 An Azure account is required for deployment. [Terraform Cloud](https://app.terraform.io) is used to store the [infrastructure state remotely](https://www.terraform.io/docs/state/remote.html). This is optional, but recommended.
 
-*This configuration will currently deploy the `symposium2019` branch of the [CCPPETMR VM](https://github.com/CCPPETMR/CCPPETMR_VM).*
+*This configuration will currently deploy the `symposium2019` branch of the [SynerBI VM](https://github.com/SynerBI/SynerBI_VM).*
```
Comment (casperdcl) [inline cloud/terraform/azure/README.md:6]: assumes VM repo rename

File: cloud/terraform/azure/packer/scripts/provision.sh [hunk 5]
```diff
@@ -9,15 +9,15 @@ sudo apt-get -y upgrade
...
 
-cd CCPPETMR_VM
+cd SynerBI_VM
 sudo bash scripts/INSTALL_prerequisites_with_apt-get.sh
 sudo bash scripts/INSTALL_python_packages.sh
 sudo apt-get purge cmake -y
 sudo bash scripts/INSTALL_CMake.sh
 
-cd ~/devel/CCPPETMR_VM
+cd ~/devel/SynerBI_VM
```
Comment (casperdcl) [inline cloud/terraform/azure/packer/scripts/provision.sh:20]: assumes VM repo rename

File: docker/Dockerfile (hunk 3 above)
Comment (casperdcl) [reply docker/Dockerfile:7]: idk; I mentioned this as something to check (in the checklist)

File: README.md [hunk 6]
```diff
@@ -53,7 +53,7 @@ mkdir ~/devel
 ```
 
 ### Install CMake
-If you do not have CMake >= 3.10, install it first. You can probably use a package manager on your OS. Alternatively, you can do that either by following the official instructions ([download link](https://cmake.org/download/)) or running your own shell sript to do so (see an example [here](https://github.com/CCPPETMR/CCPPETMR_VM/blob/master/scripts/INSTALL_CMake.sh)). 
+If you do not have CMake >= 3.10, install it first. You can probably use a package manager on your OS. Alternatively, you can do that either by following the official instructions ([download link](https://cmake.org/download/)) or running your own shell sript to do so (see an example [here](https://github.com/SynerBI/SynerBI_VM/blob/master/scripts/INSTALL_CMake.sh)).
```
Comment (casperdcl) [inline README.md:56]: assumes VM repo rename

File: cloud/terraform/azure/README.md (hunk 4 above)
Comment (KrisThielemans) [reply cloud/terraform/azure/README.md:6]: done

**Comment by casperdcl** [comment]
https://travis-ci.org/github/SynerBI/SIRF-SuperBuild/builds/672550235 passed but https://travis-ci.org/github/SynerBI/SIRF-SuperBuild/builds/672660623 failed :/

**Comment by casperdcl** [comment]
Just a quick explicit confirmation... @KrisThielemans is it `SynerBI` or `SyneRBI`? It looks like the gh org was created with the former...

**Comment by KrisThielemans** [comment]
sigh. SyneRBI. Not sure what a github rename will do, as https://github.com/SyneRBI/ works as well. Just rename again?

**Comment by casperdcl** [comment]
Sure, I think renaming again should be fine. URLs are case-insensitive. The main thing that would change is the email notification subject lines.

**Comment by KrisThielemans** [comment]
I've renamed the org. So now we need to correct all @paskino 's renames (e.g. in the headers). @paskino, could you take care of this?

**Comment by paskino** [comment]
Sure, my bad, I'll do that.

**Comment by paskino** [comment]
I've updated the Superbuild headers, however:

1. can we make without the `author`? It'd make things way easier to maintain. Some headers don't have any header, let alone `author`. I suggest to keep an `authors.md` file listing all contributors.
1. I'll try to update all the references to `SynerBI` to `SyneRBI`, such the project name in the CMake files...
1. there are references to the [www.synerbi.ac.uk](https://www.synerbi.ac.uk) here and there but it doesn't yet exists and the reference to the [ccppetmr ](https://www.ccppetmr.ac.uk) website has been removed.

**Comment by KrisThielemans** [comment]
> can we make without the author? 

It's indeed a pain. However, I've noticed that some contributors really like to have their name listed in the file that is changed. On the other hand, others can't be bothered to update the header.

In any case, the author isn't the only a problem, more important is the copyright owner. That is often forgotten as well. However, I can't see how we can dispense with that. This is legal information, that otherwise would only be visible via `git` (if we assume that we never commit on behalf of someone else), and it wouldn't be visible on `conda` or so.

As we need to maintain the copyright message, we can just as well add the author as well...

**Comment by KrisThielemans** [comment]
Re the website, either you undo those changes, or we wait a bit for merging. I think I'd choose the latter...

**Comment by paskino** [comment]
Let's wait ;)

File: CMake/FindSPM.cmake [hunk 7]
```diff
@@ -1,18 +1,22 @@
...
 #
-# This file is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Lesser General Public License for more details.
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```
Comment (KrisThielemans) [inline CMake/FindSPM.cmake:17]: @rijobro why did you use LGPL here? copied from somewhere? If so, we have to stick to it

File: CMake/RenameVariable.cmake [hunk 8]
```diff
@@ -1,8 +1,8 @@
 #========================================================================
 # Author: Kris Thielemans
-# Copyright 2019 University College London
+# Copyright 2019, 2020 University College London
 #
-# This file is part of the CCP PETMR Synergistic Image Reconstruction Framework (SIRF) SuperBuild.
+# This file is part of the CCP SyneRBI (formely PETMR) Synergistic Image Reconstruction Framework (SIRF) SuperBuild.
```
Comment (KrisThielemans) [inline CMake/RenameVariable.cmake:5]:
```suggestion
# This file is part of the CCP SyneRBI (formerly PETMR) Synergistic Image Reconstruction Framework (SIRF) SuperBuild.
```

File: CMake/SetExternalProjectFlags.cmake [hunk 9]
```diff
@@ -2,7 +2,7 @@
 # Author: Ander Biguri
 # Copyright 2020 University College London
 #
-# This file is part of the CCP PETMR Synergistic Image Reconstruction Framework (SIRF) SuperBuild.
+# This file is part of the CCP SyneRBI (formely PETMR) Synergistic Image Reconstruction Framework (SIRF) SuperBuild.
```
Comment (KrisThielemans) [inline CMake/SetExternalProjectFlags.cmake:5]:
```suggestion
# This file is part of the CCP SyneRBI (formerly PETMR) Synergistic Image Reconstruction Framework (SIRF) SuperBuild.
```
please check all the rest

File: SuperBuild/External_ACE.cmake [hunk 10]
```diff
@@ -74,7 +74,7 @@ if(NOT ( DEFINED "USE_SYSTEM_${externalProjName}" AND "${USE_SYSTEM_${externalPr
   )
   endif()
 
-# Currently, setting ACE_ROOT has no effect, see https://github.com/CCPPETMR/SIRF-SuperBuild/issues/147
+# Currently, setting ACE_ROOT has no effect, see https://github.com/SynerBI/SIRF-SuperBuild/issues/147
```
Comment (KrisThielemans) [inline SuperBuild/External_ACE.cmake:77]:
```suggestion
# Currently, setting ACE_ROOT has no effect, see https://github.com/SyneRBI/SIRF-SuperBuild/issues/147
```

File: docker/Dockerfile (hunk 3 above)
Comment (KrisThielemans) [reply docker/Dockerfile:7]: @paskino?

File: version_config.cmake [hunk 11]
```diff
@@ -50,7 +49,7 @@ set(Armadillo_MD5 c2fa488ea069b9972363ebad16e51ab5 )
 if(WIN32)
   # Just use precompiled version
   # TODO would prefer the next zip file but for KT using an ftp URL times-out (firewall?)
-  set(FFTW3_URL https://www.ccppetmr.ac.uk/sites/www.ccppetmr.ac.uk/files/downloads/fftw-3.3.5-dll64.zip )
+  set(FFTW3_URL https://www.ccpsynerbi.ac.uk/sites/www.ccpsynerbi.ac.uk/files/downloads/fftw-3.3.5-dll64.zip )
```
Comment (KrisThielemans) [inline version_config.cmake:52]: not on assets?

File: docker/Dockerfile (hunk 3 above)
Comment (paskino) [reply docker/Dockerfile:7]: What? Is it going to be me? That's what you want to say @KrisThielemans ?

Comment (casperdcl) [reply docker/Dockerfile:7]: I think the question is have you created synerbi@stfc.ac.uk or cppsynerbi@stfc.ac.uk etc?

File: version_config.cmake (hunk 11 above)
Comment (casperdcl) [reply version_config.cmake:52]:
```suggestion
  set(FFTW3_URL https://github.com/SyneRBI/assets/releases/download/latest/fftw-3.3.5-dll64.zip )
```

**Comment by casperdcl** [comment]
@paskino I already made those changes 9h ago :) did you mean to merge, btw?

**Comment by paskino** [comment]
@casperdcl I just added a few changes to comments, nothing functional, as per https://github.com/SyneRBI/SIRF-SuperBuild/pull/372#pullrequestreview-399727265
After this merging was the plan, afaik.

**Pull request merged**